ocm/scripts/watcher-ocm-create.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+#Creates OCM package using watcher helm chart
+
+REPO_ROOT=../..
+MODULE_NAME="kyma-project.io/module/kyma-watcher/operator"
```

Comment:
this needs correction
```suggestion
MODULE_NAME="kyma-project.io/module/kyma-watcher/skr"
```